From: NOKUBI Takatsugu <knok@daionet.gr.jp>
Date: Wed, 23 Jan 2019 14:29:56 +0900
Subject: work with emacs 24.3.1

---
 src-util/anthy.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src-util/anthy.el b/src-util/anthy.el
index 9f0e225..5eb16d5 100644
--- a/src-util/anthy.el
+++ b/src-util/anthy.el
@@ -892,7 +892,7 @@
 	 ((event-matches-key-specifier-p event 'backspace) 8)
 	 (t
 	  (char-to-int (event-to-character event)))))
-    last-command-char))
+    last-command-event))
 
 ;;
 ;;
