From: NOKUBI Takatsugu <knok@daionet.gr.jp>
Date: Sat, 20 Apr 2019 16:30:04 +0900
Subject: new Japanese era reiwa entry

---
 alt-cannadic/gcanna.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/alt-cannadic/gcanna.t b/alt-cannadic/gcanna.t
index 960ebb9..43927fc 100644
--- a/alt-cannadic/gcanna.t
+++ b/alt-cannadic/gcanna.t
@@ -134761,7 +134761,7 @@
 れいれいしめ #T15*10 麗々しめ 麗麗しめ
 れいれいしゅう #KYU*10 麗々しゅう 麗麗しゅう
 れいろう #F02*200 玲瓏
-れいわ #T35*150 例話
+れいわ #T35*200 令和 #T35*150 例話
 れいをい #W5*200 礼を言
 れいん #T35*300 レイン #JN*100 レイン
 れいんうぇあ #T35*250 レインウェア
